library/include/ck/library/reference_tensor_operation/cpu/reference_batchnorm_forward_nhwc_c.hpp

```diff
@@ -68,32 +74,30 @@ struct ReferenceBatchNormFwd_Input_N_H_W_C_Output_C : public device::DeviceBatch
         const AccDataType* bnBias_;
         InOutDataType* p_y_;
 
-        AccDataType* resultRunningMean_;
-        AccDataType* resultRunningVariance_;
         AccDataType* resultSaveMean_;
         AccDataType* resultSaveInvVariance_;
+        AccDataType* resultRunningMean_;
```

Comment:
We should not assume scale and bias be AccDataType.
Separate scale, bias, mean and variance datatype.
I found you separate them in device op.

Reply:
Fixed in [e2c66](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/403/commits/ad3db30feb22fbcc98c949471be30247ffde2c66)